test/Game-test.js

```diff
@@ -23,10 +24,45 @@ describe('it should create a game ', function() {
   })
 
   it('it should start a game', function() {
-    
-    game.start()
+    game.start();
     expect(domUpdates.getPlayerNames).to.have.been.called(1);
+    expect(domUpdates.hideStartScreen).to.have.been.called(1);
+    expect(game.currentWheel).to.have.property('elements').with.lengthOf(6);
+    expect(game).to.have.property('players').with.lengthOf(3);
+    expect(game).to.have.property('puzzles').with.lengthOf(4);
+    expect(game.currentPuzzle).to.be.an.instanceOf(Puzzle);
+    expect(domUpdates.displayPuzzle).to.have.been.called(1);
+  });
+
+  it('it should instanciate three players from Player class', function() {
+    let playerNames = ['Hank', 'Dustin', 'Katiebelle'];
+    game.createPlayers(playerNames);
+    expect(game.players).to.have.lengthOf(3);
+    expect(game.players[0, 1, 2]).to.be.an.instanceOf(Player);
+    expect(game.players[0].name).to.equal('Hank');
   });
 
+  it('should return an array of puzzles', function() {
+    let bankOfPuzzles = game.createPuzzleBank();
+    expect(bankOfPuzzles).to.be.instanceOf(Array);
+    expect(bankOfPuzzles).to.have.lengthOf(96);
+    expect(bankOfPuzzles[0]).to.be.instanceOf(Object);
```

Comment:
Can you be more specific here than `Object` (I'm not sure if you can - just asking because I haven't looked at the implementation)?